src/coreclr/src/inc/clrconfigvalues.h

```diff
@@ -383,7 +383,7 @@ RETAIL_CONFIG_DWORD_INFO(INTERNAL_InterpreterMethHashMin, W("InterpreterMethHash
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_InterpreterMethHashMax, W("InterpreterMethHashMax"), UINT32_MAX, "If non-zero, only interpret methods selected by 'Interpret' whose hash is at most this value")
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_InterpreterStubMin, W("InterpreterStubMin"), 0, "Only interpret methods selected by 'Interpret' whose stub num is at least this value.")
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_InterpreterStubMax, W("InterpreterStubMax"), UINT32_MAX, "If non-zero, only interpret methods selected by 'Interpret' whose stub number is at most this value.")
-RETAIL_CONFIG_DWORD_INFO(INTERNAL_InterpreterJITThreshold, W("InterpreterJITThreshold"), 10, "The number of times a method should be interpreted before being JITted")
+RETAIL_CONFIG_DWORD_INFO(INTERNAL_InterpreterJITThreshold, W("InterpreterJITThreshold"), 10, "The number of times (in hexadecimal) a method should be interpreted before being JITted")
```

Comment:
It's fine to add this clarification, but note that _every_ integer COMPlus value is parsed as hex, not just this one.

Reply:
Hi @BruceForstall, thanks for mentioning this! Maybe this is specified somewhere in the documentation but I missed it. Anyway, I remove this redundant annotation in commit https://github.com/dotnet/runtime/pull/37825/commits/2328ced54e384af01f53769998dd0ed957167996.